mesonbuild/mintro.py

```diff
@@ -189,12 +189,14 @@ def list_buildoptions(coredata: cdata.CoreData) -> List[Dict[str, Union[str, boo
                         'infodir',
                         'libdir',
                         'libexecdir',
+                        'libpkgconfigdir',
```

Comment:
`dir_option_names` in mconf.py also has to be updated.

Reply:
The patch already includes `dir_option_names` change for mesonbuild/mconf.py, so I don't have to do anything for this comment, right?

Reply:
Yes, this is resolved now.